python/mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+strict_optional = False
```

Comment:
We should enable this ASAP, but there are too many violations right now.

Reply:
I agree